resources/views/profile.blade.php

```diff
@@ -55,8 +55,22 @@
                             />
                         @endforeach
                     </x-section>
-                </div>
-            @endif
+                @else
+                    <div class="flex flex-col items-start gap-2">
+                        <p class="text-slate-200">
```

Comment:
I think the component <x-empty-state /> is merged now 
```suggestion
                        <x-empty-state content="Looks like your list section is empty. Time to
                            create your first list!" />
```